src/modules/0.0.21/iasql_functions/iasql.ts

```diff
@@ -897,21 +899,17 @@ export async function upgrade(dbId: string, dbUser: string, context: Context) {
     if (!db) return 'Database no found (somehow)';
     await MetadataRepo.dbUpgrading(db, true);
     (async () => {
+      // We need to sleep for a bit to let the scheduler response execute before we start messing up with the db
+      await new Promise(r => setTimeout(r, 5000));
```

Comment:
would it be possible to explicitly wait on something here or no? just mindful of introducing flakiness if the schedule is not ready, but I understand this is the nature of upgrade if there is nothing else

Reply:
I get your point but I do not think there's anything else we can do here more than wait.